File: GFG/Fitting The Array/Fitting The Array.cpp
class Solution{
public:
    
    bool isFit(int arr[], int brr[], int n){
        // code here
        sort(arr, arr+n);
        sort(brr, brr+n);
        
        for(int i=0; i<n; i++)
            if(arr[i] > brr[i])
                return false;
                
        return true;
    }
};

File: GFG/Fitting The Array/Fitting The Array.java
class Solution{
    
   
    // Function for finding maximum and value pair
    public static boolean isFit (int arr[], int brr[], int n) {
        //Complete the function
        Arrays.sort(arr);
        Arrays.sort(brr);
        for(int i=0;i<n;i++){
            if(arr[i]>brr[i]){
                return false;
            }
        }
        return true;
    }
    
}
